caas/kubernetes/provider/specs/v3_test.go

```diff
@@ -308,47 +308,56 @@ kubernetesResources:
                     serviceName: test
                     servicePort: 80
   mutatingWebhookConfigurations:
-    example-mutatingwebhookconfiguration:
-      - name: "example.mutatingwebhookconfiguration.com"
-        failurePolicy: Ignore
-        clientConfig:
-          service:
-            name: apple-service
-            namespace: apples
-            path: /apple
-          caBundle: "YXBwbGVz"
-        namespaceSelector:
-          matchExpressions:
-            - key: production
-              operator: DoesNotExist
-        rules:
-          - apiGroups:
-              - ""
-            apiVersions:
-              - v1
-            operations:
-              - CREATE
-              - UPDATE
-            resources:
-              - pods
+    - name: example-mutatingwebhookconfiguration
```

Comment:
We'll have to communicate this change to the k8s folks etc

Reply:
yeah, v3 is introduced in 2.8 recently.
So it should be ok to make this change without upgrading the version.
But I will update the v3 change document.